src/ApiStatus.ts

```diff
@@ -104,22 +115,22 @@ export default class ApiStatus {
     }
 
     private parseApiStatus(apiStatus: APIStatus): StatusEmbedState {
-        let cacheObject: { [key: string]: any } = {};
+        const cacheObject: { [key: string]: any } = {};
 
         let onFire = false;
         let allApisOK = true;
         let allApisIssues = false;
         let apiIssuesCounter = 0;
         let apiCounter = 0;
-        let statusEmbed: StatusEmbedState = { api: {}, onFire: false, allApisOK: true, allApisIssues: false};
+        const statusEmbed: StatusEmbedState = { api: {}, onFire: false, allApisOK: true, allApisIssues: false};
 
-        type RegionState = {"troubled": Array<string>, "up": Array<string>, "down": Array<string>};
+        type RegionState = {"troubled": string[], "up": string[], "down": string[]};
```

Comment:
It might be better to move this to the top level with the other type declarations. You can also remove the quotes from the key names, with the same rationale as in the previous review.

Reply:
I will remove the quotes and move it into line 130 (which is the only place where this is used).